packages/@aws-cdk/core/lib/app.ts

```diff
@@ -35,11 +35,20 @@ export interface AppProps {
   readonly stackTraces?: boolean;
 
   /**
-   * Include runtime versioning information in cloud assembly manifest
-   * @default true runtime info is included unless `aws:cdk:disable-runtime-info` is set in the context.
+   * Include runtime versioning information in the Stacks of this app
+   *
+   * @deprecated use `versionReporting` instead
+   * @default Value of 'aws:cdk:version-reporting' context key
    */
   readonly runtimeInfo?: boolean;
 
+  /**
+   * Include runtime versioning information in the Stacks of this app
+   *
+   * @default Value of 'aws:cdk:version-reporting' context key
+   */
+  readonly versionReporting?: boolean;
```

Comment:
I am wondering if the correct terminology should be `analytics: boolean`--broaden the scope a little bit.